manifests/kots-helm.yaml

```diff
@@ -104,7 +104,7 @@ spec:
       awsS3Region: '{{repl ConfigOption "storageBucketsAwsS3Region" }}'
       importAwsRoleArn: '{{repl ConfigOption "storageBucketsImportExportAwsArnRole" }}'
```

Comment:
This fields aren't properly configured, right? If we set the value but we hide the option in the config, the value keep placed here, right? :thinking:

Reply:
The point here is... If someone configure it in Kots config and then disable the option (so is hidden), the value keep returned and setted here, right? That's not a desired behaviour.

I'm not entirely sure if it works that way, that's why I'm asking you. I think it does since that's why we are using full conditional blocks.

Reply:
You're totally right @ManuelLR 😄 I'm pushing the change